src/server.c

```diff
@@ -622,6 +622,10 @@ struct redisCommand redisCommandTable[] = {
      "write fast @keyspace",
      0,NULL,1,1,1,0,0,0},
 
+    {"copy",copyCommand,-3,
+     "write fast @keyspace",
```

Comment:
+use-memory

Reply:
thanks @madolson ! I've fixed the flag in the command. I also fixed the copy command a bit.

But the code doesn't pass the test.
What do I need to do to make sure my code passes the test?

Also, should I combine my commits into one?

Reply:
i guess `fast` should be removed? considering some types may require O(N) loops?

also, not sure about `@keyspace`
the only hint in the docs is:
```
 * 7) The non obvious "keyspace" category includes the commands
 *    that interact with keys without having anything to do with
 *    specific data structures, such as: DEL, RENAME, MOVE, SELECT,
 *    TYPE, EXPIRE*, PEXPIRE*, TTL, PTTL, ...
```
so maybe i'd consider COPY similar to BRPOPLPUSH?
i guess not (we should keep `@keyspace`), but wanted to raise this concern to see if anyone has any feedback.

Reply:
I think we should keep @keyspace